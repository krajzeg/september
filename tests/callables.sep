# Implement a new Array.<call> on our own.

Array:"<call>" = |...elements| { elements }

print("This should list 1,2,3 in order.")
for (number) in (Array(1,2,3)) {
	print(number)
}

