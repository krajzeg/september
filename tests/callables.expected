This shouldn't print anything.
This should print 'Woof!' twice.
Woof!
Woof!
